_drafts/2017-xx-xx-patentsview.Rmd

```diff
@@ -0,0 +1,244 @@
+
+Combined, these two things make patents a prime target for data analysis. However, until recently it was hard to get at the data inside these documents. One had to either collect it manually using USPTO’s search engine, or figure out a way to download, parse, and model huge XML data dumps. Enter PatentsView.
+
+## PatentsView and `patentsview`
+
+[PatentsView](http://www.patentsview.org/web/#viz/relationships) is one of the [United States Patent and Trademark Office](https://en.wikipedia.org/wiki/United_States_Patent_and_Trademark_Office)'s new initiatives intended to increase the usability and value of patent data. One feature of this project is a publicly accessible API that makes it easy to programmatically interact with the data. A few of the reasons why I like the API (and PatentsView more generally):
+
+<!-- TO REVIEWER: Do you think I should drop the discussion of why i like patentsview? -->
+
+* The API is free (no credential required) and currently doesn't impose rate limits/bandwidth throttling.
+* The project offers [bulk downloads of patent data](http://www.patentsview.org/download/) on their website (in a flat file format), for those who want to be closest to the data. 
+* Both the API and the bulk download data contain disambiguated entities such as inventors, assignees, organizations, etc. In other words, the API will tell you whether it thinks that John Smith on patent X is the same person as John Smith on patent Y.[^1]  
+
+The `patentsview` R package is a wrapper around the PatentsView API. It contains a function that acts as a client to the API (`search_pv()`) as well as several supporting functions. Full documentation of the package can be found on its [website](https://ropensci.github.io/patentsview/index.html).
+
+## Installation
+
+You can install the stable version of `patentsview` from CRAN:
+
+```{r, eval = FALSE}
+install.packages("patentsview")
+```
+
+Or get the development version from GitHub:
+
+```{r eval = FALSE}
+if (!require(devtools)) install.packages("devtools")
+
+devtools::install_github("ropensci/patentsview")
+```
+
+## Getting started
+
+The package has one main function, `search_pv()`, that makes it easy to send requests to the API. There are two parameters to `search_pv()` that you're going to want to think about just about every time you call it - `query` and `fields`. You tell the API how you want to filter the patent data with `query`, and which fields you want to retrieve with `fields`.[^2] 
+
+### `query`
+
+Your query has to use the [PatentsView query language](http://www.patentsview.org/api/query-language.html), which is a JSON-based syntax that is similar to the one used by Lucene. You can write the query directly and pass it as a string to `search_pv()`:
```

Comment:
add a relevant link to Lucene to show why you made the comparison (for readers not familiar)? or is that a diversion?

Reply:
I think it's somewhat of a diversion. Can we not link to Lucene here?